drivers/amazonec2/amazonec2.go

```diff
@@ -903,15 +889,13 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) []*
 	hasDockerPort := false
 	hasSwarmPort := false
 	for _, p := range group.IpPermissions {
-		if p.FromPort != nil {
```

Comment:
yeah,  was hoping to get the open issue fixed before pushing a fix.

On Fri, Jan 15, 2016 at 2:16 PM, Nathan LeClaire <notifications@github.com>
wrote:

> In drivers/amazonec2/amazonec2.go
> <https://github.com/docker/machine/pull/1956#discussion_r49892979>:
>
> > @@ -903,15 +889,13 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) []*
> >  	hasDockerPort := false
> >  	hasSwarmPort := false
> >  	for _, p := range group.IpPermissions {
> > -		if p.FromPort != nil {
>
> Hm, looks like there's a few issues with rebase, so I'm guessing this
> wasn't intentional.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/machine/pull/1956/files#r49892979>.
>